package.json

```diff
@@ -7,7 +7,9 @@
     "Karl Horky <karl.horky@gmail.com>"
   ],
   "license": "MIT",
-  "main": "dist/index.js",
+  "main": "dist/preflight.esm.js",
+  "module": "dist/preflight.esm.js",
+  "type": "module",
```

Comment:
It seems execa need type module to work with TSDX

Reply:
Looks like the reason may be this: when executing the `bin/preflight.js` file, it fails because it doesn't know that we want to use ESM.

<img width="848" alt="Screen Shot 2021-04-23 at 11 14 28" src="https://user-images.githubusercontent.com/1935696/115849622-59118e00-a425-11eb-833f-18aa2cbda532.png">